includes/services.inc.php

```diff
@@ -129,7 +129,7 @@ function poll_service($service)
 
     // If we do not have a cmd from the check script, build one.
     if ($check_cmd == "") {
-        $check_cmd = $config['nagios_plugins'] . "/check_" . $service['service_type'] . " -H " . ($service['service_ip'] ? $service['service_ip'] : $service['hostname']);
```

Comment:
It's not needed for the checks, because it makes errors for plugins (nagios and others). Some plugins work, some don't. 

For example, if you use it 
"/check_" . $service['service_type'] . " -H " . ($service['service_ip'] ? $service['service_ip'] : $service['hostname']);

It runs "-H" by default. But some plugins don't need "-H" in command line, and you get "syntax error"

1. Check APC
![apc_plugin_01](https://user-images.githubusercontent.com/17820730/27407615-55dbc966-5696-11e7-9d93-e8c6435e1d60.jpg)
![apc_plugin_02](https://user-images.githubusercontent.com/17820730/27407710-b6ab5b44-5696-11e7-9863-0bdc071728dd.jpg)

2. Check Mitel
![mitel_plugin](https://user-images.githubusercontent.com/17820730/27407808-296045aa-5697-11e7-8680-e19b31a63f20.jpg)
![mitel_plugin_02](https://user-images.githubusercontent.com/17820730/27407897-7b06ed50-5697-11e7-86cd-c2192c6a08ee.jpg)

3. Another strange thing is when you want to use "ping" from "Nagios plugins".
the normal command for ping is: 
-H 172.20.1.56 -w 1000.0,20% -c 2000.0,60%
but if you run it, it looks like:
**-H** -H 172.20.1.56 -w 1000.0,20% -c 2000.0,60%
and result will be:
![ping_plugin](https://user-images.githubusercontent.com/17820730/27408175-a2288e38-5698-11e7-9719-a6d181a31bbe.jpg)

But what is it? Is it normal result? It's garbage!!!

If you use "/check_" . $service['service_type'] ;  , I mean without "-H"
And type normal Parameters (which usually we use from http://nagios-plugins.org/doc/man/check_ping.html):
![ping_plugin_02](https://user-images.githubusercontent.com/17820730/27408453-f659e082-5699-11e7-831b-a98dc9f02826.jpg)

you will get true result, which must be:
![ping_plugin_03](https://user-images.githubusercontent.com/17820730/27408693-2672fa00-569b-11e7-91cc-48533fb43f9a.jpg)

If you need to run service, please, type full command from plugin documentation.
I hope you understand what I mean!